### ISSUE TITLE
Build instructions not working

### ISSUE THREAD

**Issue by pgundlach** [HOST-Oman/libraqm#91]
I've cloned the repository but I can't build it. There is no `configure` script to run. 

```
$ ./configure
$ make
$ make install
```

I guess there is something missing.

When I run `./autogen.sh`, I get

```
-n checking for gtkdocize... 
/usr/local/bin/gtkdocize
-n checking for autoreconf... 
/usr/local/bin/autoreconf
running gtkdocize --copy
running autoreconf --force --install --verbose
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal -I m4 --output=aclocal.m4t
Can't exec "aclocal": No such file or directory at /usr/local/Cellar/autoconf/2.69/share/autoconf/Autom4te/FileUtils.pm line 326.
autoreconf: failed to run aclocal: No such file or directory
```

What can I do?

**Comment by khaledhosny** [comment]
Are you able to build other projects using autotools? Looks like a generic autotools issue. You might also build from release tarballs where you don’t need to run `autogen.sh`.

**Comment by pgundlach** [comment]
Hello Khaled, thanks for the quick answer. I was missing the `automake` formula from homebrew (Mac). Now `./autogen.sh` works fine.

It would have helped me to have a notice such as 

```
$ ./autogen.sh
$ ./configure
$ make
$ make install
```

But I don't know it this makes sense or not. (And I should have searched the web before posting the error message above).

**Comment by khaledhosny** [comment]
Yes, you need `automake`,  `autoconf` and may be others when building from Git repository. If you can suggest any updates to the README, I can incorporate them.

**Comment by pgundlach** [comment]
Well, I would just add the line with `$ ./autogen.sh`, but since I am absolutely not an export using autotools, I leave this to you. I don't have any other enhancements for the Readme (for now ;-))

**Comment by khaledhosny** [comment]
Building from release tarballs does not require running `autogen.sh` since the `configure` script is bundled, that is why it isn’t there since building from tarballs is usually the recommended method for projects using autotools.
